devel/idevicerestore/Portfile

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        libimobiledevice idevicerestore 1.0.0
+
+categories          devel
+platforms           darwin
+
+license             LGPL-3
+maintainers         {@i0ntempest me.com:szf1234} openmaintainer
+
+description         The libirecovery library allows communication with iBoot/iBSS of iOS devices via USB.
+
+long_description    The idevicerestore application is a full reimplementation of all granular steps which \
+                    are performed during the restore of a firmware to a device. In general, upgrades and \
+                    downgrades are possible, however subject to availability of SHSH blobs from Apple for \
+                    signing the firmare files. Some key features are:\n\
+                    * Restore: Update firmware on iOS devices\n\
```

Comment:
There will be a space before each asterisk. I usually avoid that by putting the `\n` right before the asterisk, rather than at the end of the preceding line, i.e.:

```
                    signing the firmare files. Some key features are: \
                    \n* Restore: Update firmware on iOS devices \
```

Also, there's a typo there: "firmare" should be "firmware".